tests/conftest.py

```diff
@@ -34,7 +44,7 @@ def minimal_input_record():
     """Minimal record data as dict coming from the external world."""
     return {
         "_access": {"metadata_restricted": False, "files_restricted": False},
-        "_owners": [1],
+        "owners": [1],
```

Comment:
Yes, this is a leftover...I was testing the https://github.com/inveniosoftware/invenio-records-permissions/blob/master/invenio_records_permissions/generators.py#L115 but it should be changed because the record schema has changed ;)